client/my-sites/people/invite-people/index.jsx

```diff
@@ -139,6 +140,26 @@ export default React.createClass( {
 		} );
 	},
 
+	isSubmitDisabled() {
+		const { errors, success, usernamesOrEmails } = this.state;
+		const invitees = Array.isArray( usernamesOrEmails ) ? usernamesOrEmails : [];
+
+		// If there are no invitees, then don't allow submitting the form
+		if ( this.state.sendingInvites || ! invitees.length ) {
+			return true;
+		}
+
+		if ( errors && errors.length ) {
+			return true;
+		}
+
+		// If there are invitees, and there are no errors, let's check
+		// if there are any pending validations.
+		return some( usernamesOrEmails, ( value ) => {
```

Comment:
I think this would read clearer if we use [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) instead of `some`.